README.md

```diff
@@ -125,6 +125,11 @@ General guidelines:
    where `DATASET_GROUP_NAME` is the dataset nickname provided in the `yaml` file, e.g. `DYJetsToLL_M-50-madgraphMLM_ext1`
 
 
-	 
+## Running with ParticleNET
+
+In order to produce NanoAOD along with output branches from ParticleNET (which will be found in the "Jet" and "FatJet" collections for AK4 and AK8 outputs, respectively), the following additions are needed before executing the code:
+* Follow the installation recipe in the Readme [here](https://gitlab.cern.ch/rgerosa/particlenetstudiesrun2/-/tree/master/)
+* Then copy the PNET models into place where they're needed, according to the first section in the ReadMe [here](https://gitlab.cern.ch/rgerosa/particlenetstudiesrun2/-/tree/master/TrainingNtupleMakerAK4) and [here](https://gitlab.cern.ch/rgerosa/particlenetstudiesrun2/-/tree/master/TrainingNtupleMakerAK8).
+* Then it will only work locally however. To make it work for jobs submitted to crab, put the same files in `RecoBTag/Combined/data/` and `RecoTauTag/data/` inside the CMSSW_BASE/src/ directory.
```

Comment:
can it all be done automatically in `env.sh`?

Reply:
Yes, I added everything in `env.sh`, but I was initially cautious about doing that, in case some steps might change in the future

Reply:
yes, but then we can change it on our side as well. IMO, having implemented it in the code reduces probability of errors during the setup.